upnext/tracks.py

```diff
@@ -73,12 +73,10 @@ def upvote_track(track_info, party, voter_name):
 
         up_track.save()
         voter.up_tracks.add(up_track)
-
         ordered_new = party.track_set.order_by('-score')
         new_position = get_index(up_track, ordered_new)
 
         reorder_playlist(party, old_position, new_position)
-
     else:
```

Comment:
Lol did you deliberately remove lines in the middle of `upvote` but not `downvote`?